<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_Exit"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Text"/>
    <addaction name="actionImport_Data"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionNew_Item"/>
    <addaction name="actionEdit_Item"/>
    <addaction name="actionDelete_Items"/>
    <addaction name="actionDuplicate_Selected"/>
    <addaction name="actionDelete_Selected"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionCalendar"/>
    <addaction name="actionWunderlist"/>
    <addaction name="actionEmail"/>
    <addaction name="actionPlain_Text"/>
    <addaction name="actionLine_Text"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionPrint_Settings"/>
    <addaction name="actionFilter_Settings"/>
    <addaction name="actionOpen_Map"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionSave_User_Settings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuExport"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuUser"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionNew_Item">
   <property name="text">
    <string>New Item</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEdit_Item">
   <property name="text">
    <string>Edit Item</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDelete_Items">
   <property name="text">
    <string>Delete Items</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCalendar">
   <property name="text">
    <string>Calendar</string>
   </property>
  </action>
  <action name="actionWunderlist">
   <property name="text">
    <string>Wunderlist</string>
   </property>
  </action>
  <action name="actionEmail">
   <property name="text">
    <string>Email</string>
   </property>
  </action>
  <action name="actionPlain_Text">
   <property name="text">
    <string>Plain Text</string>
   </property>
  </action>
  <action name="actionLine_Text">
   <property name="text">
    <string>Line Text</string>
   </property>
  </action>
  <action name="actionImport_Text">
   <property name="text">
    <string>Import Text</string>
   </property>
  </action>
  <action name="actionImport_Data">
   <property name="text">
    <string>Import Data</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionPrint_Settings">
   <property name="text">
    <string>Print Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionFilter_Settings">
   <property name="text">
    <string>Filter Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionDuplicate_Selected">
   <property name="text">
    <string>Duplicate Selected</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSave_User_Settings">
   <property name="text">
    <string>Save User Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionDelete_Selected">
   <property name="text">
    <string>Delete Selected</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionOpen_Map">
   <property name="text">
    <string>Open Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSave_Exit">
   <property name="text">
    <string>Save Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
